common/cards/attach/lightning-rod.ts

```diff
@@ -47,20 +47,21 @@ const LightningRod: Attach = {
 				if (attack.target?.player.entity !== player.entity) return
 
 				attack.redirect(component.entity, component.slot.row?.entity)
-			},
-		)
 
-		observer.subscribeWithPriority(
-			game.hooks.beforeAttack,
-			beforeAttack.REACT_TO_DAMAGE,
-			(attack) => {
-				if (attack.player.entity !== opponentPlayer.entity) return
-				if (!component.slot?.onBoard() || !component.slot.row) return
-				if (attack.type === 'status-effect' || attack.isBacklash) return
-				if (game.currentPlayer.entity !== opponentPlayer.entity) return
-				if (attack.target?.player.entity !== player.entity) return
-				if (attack.calculateDamage() <= 0) return
-				used = true
+				observer.subscribeWithPriority(
+					game.hooks.beforeAttack,
+					beforeAttack.ATTACKS_MISS,
```

Comment:
```suggestion
					beforeAttack.REACT_TO_MISS,
```
This listener will no longer need to be added during `beforeAttack.LIGHTNING_ROD_REDIRECT` and can be un-nested